controllers/performanceprofile_controller.go

```diff
@@ -156,6 +175,36 @@ func (r *PerformanceProfileReconciler) mcpToPerformanceProfile(mcpObj client.Obj
 	return requests
 }
 
+func (r *PerformanceProfileReconciler) tunedProfileToPerformanceProfile(tunedProfileObj client.Object) []reconcile.Request {
+	node := &corev1.Node{}
```

Comment:
Did you mean getting Node here ?  not the profile itself?

Reply:
yep, the profile will not help us to fetch the relevant performance profile, but because the tuned profile has the same name as node we do not have a problem